src/components/ProjectField/TextAreaDisplay.tsx

```diff
@@ -6,6 +6,8 @@ import { ProjectFieldProps } from '.';
 
 const ProjectFieldTextAreaDisplay = ({ label, value }: ProjectFieldProps) => {
   const theme = useTheme();
+  const valueLines = value ? `${value}`.split('\n') : null;
+  const valueElements = valueLines?.flatMap((line) => [<br key='break' />, line])?.slice(1);
```

Comment:
This was the cleanest way I could think of to turn newlines into React `<br/>` objects to preserve line breaks in the text, but maybe there's a better approach.

Reply:
I looked around quickly but didn't see a built-in component to do line-break-preserving text values, but it wouldn't surprise me if there is one and I just missed it.

Reply:
You could try using the `web-components:TextField` component, there is a `display` prop to show the value instead of an input. 

[Here's an example](https://github.com/terraware/terraware-web/blob/50eb53c2a01c619f5b6acbad5120c772715ed5ad/src/scenes/ObservationsRouter/plot/index.tsx#L122-L129).

The component also has a `preserveNewlines` prop that may be needed.

Reply:
You can try copy pasting the value you have into the Storybook [here](https://web-components-1nd6ccvl0-terraformation.vercel.app/?path=/story/textfield-new--text-area&args=helperText:;display:!true) and see if it appears as you expect.